Dockerfile

```diff
@@ -21,6 +21,13 @@ RUN apt-get install -y libfreetype6 libfreetype6-dev libfontconfig1 libfontconfi
 RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /cc-test-reporter
 RUN chmod +x /cc-test-reporter
 
+ENV YARN_VERSION 1.12.3
+ENV NODE_ENV production
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
```

Comment:
FYI, I think we're going to need to consult with devops to get your desired versions of `node` and `yarn` in deployed environments.  Looks like versions are currently derived from [this file](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/build/roles/common/nodejs/vars/main.yml).

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/Dockerfile#L9 has yarn version as 1.12.3 and is using a `node:10` image as it's base image

I'm not seeing `YARN_VERSION` listed as a property in https://github.com/department-of-veterans-affairs/vets-website/blob/master/docker-compose.yml which would enable the changing of the value I believe

Reply:
@jholton is right about versions.. but that's the default for the module. We can specify newer versions, just need to keep it in sync with what's here.

Reply:
So this is going to just keep updating to 10. latest right? It looks like it's being pinned to 10.15.3 for the review instances on vets-website https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/roles/review-instance-configure/meta/main.yml

Reply:
Updated Dockerfile to install a specific version of node using nvm, currently set to 10.15.3 via NODEJS_VERSION environment variable